eea/volto/policy/restapi/services/actions/get.py

```diff
@@ -0,0 +1,54 @@
+"""Breadcrumbs"""
+
+from zope.component import getMultiAdapter
+from zope.component import adapter
+from zope.i18n import translate
+from zope.interface import implementer
+from zope.interface import Interface
+from plone.restapi.interfaces import IExpandableElement, IPloneRestapiLayer
+from plone.restapi.services.actions.get import Actions
+
+
+@implementer(IExpandableElement)
+@adapter(Interface, IPloneRestapiLayer)
+class EEAActions(Actions):
+    """EEA Actions"""
+
+    def __call__(self, expand=False):
+        """ """
+        result = {"actions": {"@id":
+                              f"{self.context.absolute_url()}/@actions"}}
+        if not expand:
+            return result
+
+        context_state = getMultiAdapter(
+            (self.context, self.request), name="plone_context_state"
+        )
+
+        categories = self.request.form.get("categories", self.all_categories)
+        filtered_action_props = {
+            "category",
+            "link_target",
+            "available",
+            "visible",
+            "allowed",
+            "modal",
+        }
+        data = {}
+        for category in categories:
+            category_action_data = []
+            actions = context_state.actions(category=category)
+            for action in actions:
+                # EEA allow actions props to be served by the endpoint except
+                # for the ones listed in filtered_action_props
+                action_data = {
```

Comment:
```Python
>>> import this
The Zen of Python, by Tim Peters

Beautiful is better than ugly.
Explicit is better than implicit.
Simple is better than complex.
Complex is better than complicated.
Flat is better than nested.
Sparse is better than dense.
Readability counts.
Special cases aren't special enough to break the rules.
Although practicality beats purity.
Errors should never pass silently.
Unless explicitly silenced.
In the face of ambiguity, refuse the temptation to guess.
There should be one-- and preferably only one --obvious way to do it.
Although that way may not be obvious at first unless you're Dutch.
Now is better than never.
Although never is often better than *right* now.
If the implementation is hard to explain, it's a bad idea.
If the implementation is easy to explain, it may be a good idea.
Namespaces are one honking great idea -- let's do more of those!
```